test/Dialect/FIRRTL/infer-resets-errors.mlir

```diff
@@ -0,0 +1,223 @@
+// RUN: circt-opt --pass-pipeline='firrtl.circuit(firrtl-infer-resets)' --verify-diagnostics --split-input-file %s
+
+// Tests extracted from:
+// - test/scala/firrtlTests/InferResetsSpec.scala
+// - test/scala/firrtl/FullAsyncResetTransform.scala
```

Comment:
It may be good to differentiate that these are coming from separate repositories and that one of them is not currently open source. Something like the following:
```suggestion
// - github.com/chipsalliance/firrtl: test/scala/firrtlTests/InferResetsSpec.scala
// - github.com/sifive/$internal:     test/scala/firrtl/FullAsyncResetTransform.scala
```